tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.6.0 (August 4, 2020)
+
+### Breaking Changes
+
+- Upgrade to `v0.7.0` of `opentelemetry` (#867)
```

Comment:
It looks like there aren't really any changes in the `tracing-opentelemetry` API as a result of this, just in the types that are passed _into_ the crate's API?

I'm not sure whether those changes should be listed here or not. Is there a changelog entry for `opentelemetry` 0.7 that we could link to, so folks can get the full list of changes?

Reply:
Yeah there are no changes here besides the examples in docs, could link to https://github.com/open-telemetry/opentelemetry-rust/blob/master/CHANGELOG.md#v070 for example?

Reply:
Let's add the link, and then I think this good.

Reply:
Done 👍